app/controllers/brexit_checker_controller.rb

```diff
@@ -12,7 +13,7 @@ class BrexitCheckerController < ApplicationController
   ]
 
   before_action :enable_caching, only: %i[show email_signup confirm_email_signup]
-  before_action :enable_caching_unless_accounts, only: %i[results]
+  before_action :enable_caching_unless_logged_in, only: %i[results]
```

Comment:
We can actually just enable caching now, as we're setting the `Vary` response header in the `set_account_variant` action.  So one user's results page won't be cached for another.

Reply:
ah ace, I did wonder about this bit.
Cool will add this to the pile with the others